apps/Standalone/src/templates/app/TemplatesStandaloneDesigner.tsx

```diff
@@ -190,69 +190,75 @@ export const TemplatesStandaloneDesigner = () => {
             isConsumption={isConsumption}
             existingWorkflowName={existingWorkflowName}
           >
-            <TemplateFilters
-              connectors={[
-                {
-                  value: 'azureaisearch',
-                  displayName: 'Azure AI Search',
-                },
-                {
-                  value: 'openai',
-                  displayName: 'Open AI',
-                },
-                {
-                  value: 'sql',
-                  displayName: 'SQL',
-                },
-                {
-                  value: 'amazon',
-                  displayName: 'Amazon',
-                },
-              ]}
-              detailFilters={{
-                Trigger: [
-                  {
-                    value: 'Request',
-                    displayName: 'Request',
-                  },
-                  {
-                    value: 'Instant',
-                    displayName: 'Instant',
-                  },
-                ],
-                By: [
-                  {
-                    value: 'Microsoft',
-                    displayName: 'Microsoft',
-                  },
-                  {
-                    value: 'Other',
-                    displayName: 'Other',
-                  },
-                ],
-                Type: [
+            <div
+              style={{
+                margin: '20px',
+              }}
+            >
+              <TemplateFilters
+                connectors={[
                   {
-                    value: 'Workflow',
-                    displayName: 'Workflow',
+                    value: 'azureaisearch',
+                    displayName: 'Azure AI Search',
                   },
                   {
-                    value: 'Other',
-                    displayName: 'Other',
+                    value: 'openai',
+                    displayName: 'Open AI',
                   },
-                ],
-                Industry: [
                   {
-                    value: 'Analytics',
-                    displayName: 'Analytics',
+                    value: 'sql',
+                    displayName: 'SQL',
                   },
                   {
-                    value: 'IT',
-                    displayName: 'IT',
+                    value: 'amazon',
+                    displayName: 'Amazon',
                   },
-                ],
-              }}
-            />
-            <TemplatesDesigner createWorkflowCall={createWorkflowCall} />
+                ]}
+                detailFilters={{
+                  Trigger: [
+                    {
+                      value: 'Request',
+                      displayName: 'Request',
```

Comment:
Same as above, we don’t want to hard code these, we should initiate them based on what is available in the templates

Reply:
I'll bring it up in the discussion, I'm not sure how that'd be done; unless we want to loop to read all templates... Maybe through a back-end call?